packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -438,6 +432,12 @@ export class ApexDebug extends DebugSession {
       case HOTSWAP_REQUEST:
         this.warnToDebugConsole(nls.localize('hotswap_warn_text'));
         break;
+      case PROXY_SETTINGS_REQUEST:
```

Comment:
Proxy settings from vscode-apex-debugger. Used in requestService.ts, streamingClient.ts, sessionService.ts, & breakpointService.ts.